<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeVisualizationView</class>
 <widget class="QWidget" name="VolumeVisualizationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_ErrorImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_NoSelectedImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Please select a volume image!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_SelectedImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="comboBoxGroup" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="m_EnableRenderingCB">
        <property name="text">
         <string>Volumerendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_RenderMode"/>
      </item>
      <item>
       <widget class="QComboBox" name="m_BlendMode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QmitkTransferFunctionGeneratorWidget" name="m_TransferFunctionGeneratorWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QmitkTransferFunctionWidget" name="m_TransferFunctionWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkTransferFunctionWidget</class>
   <extends>QWidget</extends>
   <header>QmitkTransferFunctionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QmitkTransferFunctionGeneratorWidget</class>
   <extends>QWidget</extends>
   <header>QmitkTransferFunctionGeneratorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
